drizzlepac/runastrodriz.py

```diff
@@ -430,7 +430,7 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
 
         if align_with_apriori or force_alignment or align_to_gaia:
             # Generate initial default products and perform verification
-            align_dicts = verify_alignment(_inlist,
+            align_dicts, align_table = verify_alignment(_inlist,
                                              _calfiles, _calfiles_flc,
                                              _trlfile,
```

Comment:
PEP 8 updates needed.

Reply:
Flake8 checking on runastrodriz (as I ran it) primarily returned warnings about code that has (so far) been grandfathered in; namely, camel-case variable names.   Please advise on what other PEP8/Flake8 updates need to be resolved before this can be merged.

Reply:
quality_analysis.py looks like it has some alignment issues for functions with span multiple lines due to multiple input values.